package/ios/RNCSliderComponentView.mm

```diff
@@ -118,6 +117,16 @@ - (void)RNCSendSliderEvent:(RNCSlider *)sender withContinuous:(BOOL)continuous i
 {
     float value = [sender discreteValue:sender.value];
     
+    if (value < sender.lowerLimit) {
+        value = sender.lowerLimit;
+        [sender setValue:value animated:NO];
+    }
+        
+    if (value > sender.upperLimit) {
+        value = sender.upperLimit;
+        [sender setValue:value animated:NO];
+    }
+
```

Comment:
Very similar to the Android implementation. Don't set the value when it's not within the limits.